### PR TITLE
Upgrade to v0.9.17

### LINKED ISSUE #18: Upgrade to v0.9.17
**Not required for internal testnet**

Apparently this version fixes #17, so we might want to upgrade at some point.

### PR DESCRIPTION
**Pull request by gonzamontiel**
Closes #18

### CODE REVIEW

File: node/Cargo.toml [hunk 1]
```diff
@@ -1,95 +1,102 @@
...
 name = "pendulum-node"
+repository = "https://github.com/paritytech/cumulus/"
 version = "0.1.0"
-authors = ["Satoshipay"]
-description = "Pendulum node configured with Cumulus ready"
-license = "Unlicense"
-homepage = "https://substrate.io"
-repository = 'https://github.com/substrate-developer-hub/substrate-parachain-template'
-edition = "2018"
-build = "build.rs"
```
Comment (ebma) [inline node/Cargo.toml:11]: Were all of these changes intended? Looks more like a downgrade to me

File: node/src/command.rs [hunk 2]
```diff
@@ -13,15 +13,18 @@ use sc_cli::{
...
+};
 use sp_core::hexdisplay::HexDisplay;
 use sp_runtime::traits::Block as BlockT;
 use std::{io::Write, net::SocketAddr};
 
 fn load_spec(id: &str) -> std::result::Result<Box<dyn sc_service::ChainSpec>, String> {
 	Ok(match id {
 		"dev" => Box::new(chain_spec::development_config()),
-		"pendulum-rococo" => Box::new(chain_spec::local_testnet_config()),
+		"template-rococo" => Box::new(chain_spec::local_testnet_config()),
```
Comment (ebma) [inline node/src/command.rs:27]: Was this intended? I don't know how much this matters though

**Review by ebma — APPROVED** [review]
Looks good to me.

File: node/src/command.rs (hunk 2 above)
Comment (gonzamontiel) [reply node/src/command.rs]: changed

**Pull request merged**